README.md

```diff
@@ -125,11 +125,14 @@ repositories {
 }
 
 dependencies {
-    implementation("com.github.livefront.sealed-enum:sealedenum:0.1.0")
-    kapt("com.github.livefront.sealed-enum:sealedenumprocessor:0.1.0")
+    implementation("com.github.livefront.sealed-enum:sealedenum:v0.1.0")
+    kapt("com.github.livefront.sealed-enum:sealedenumprocessor:v0.1.0")
```

Comment:
You put a `v` in the version when you tagged it didn't you?

Reply:
Yeah, that's what Bridge was doing. Do you think you will go with the leading `v` or leave it off?

Reply:
Yeah I did that because some library (Picasso?) was doing that at the time but it seems that's not the standard lately.

Reply:
I think I'll go with no `v`, leaving `v0.1.0` as a relic of the past

Reply:
As part of the `0.2.0` release, I'll change the README back.

Reply:
I also opted for the `v`less version